web/server/www/scripts/codecheckerviewer/ListOfRuns.js

```diff
@@ -101,10 +103,67 @@ function (declare, dom, ItemFileWriteStore, topic, Dialog, Button,
     return label;
   }
 
+  var RunStore = declare(Store, {
+    constructor : function (listOfRunsGrid) {
+      this.listOfRunsGrid = listOfRunsGrid;
+    },
+
+    getIdentity : function (run) {
+      return run.runId;
+    },
+
+    query : function (query, options) {
+      var deferred = new Deferred();
+      deferred.total = query.total;
+
+      var that = this;
+      CC_SERVICE.getRunData(
+        query.runFilter,
+        CC_OBJECTS.MAX_QUERY_SIZE,
+        options.start,
+        function (runDataList) {
+          if (runDataList instanceof RequestFailed)
```

Comment:
Shouldn't we use curly brackets here there is an else branch.
```suggestion
          if (runDataList instanceof RequestFailed){
          ...
          }
          else{
          ...
          }
```